pages/ChatDetails/index.tsx

```diff
@@ -1,20 +1,24 @@
 import React from 'react';
 import { ScrollView } from 'react-native';
-import ChatItem from '../../components/Chat/ChatItem';
-import { Text } from 'react-native';
+import ChatDetailHeader from '../../components/Chat/ChatShow';
+import { RouteComponentProps } from 'react-router-native';
```

Comment:
Move this up to 3rd line all the imports from npm packages should be on top.

Reply:
I have moved it